module-services/service-evtmgr/EventManager.cpp

```diff
@@ -144,6 +145,7 @@ sys::ReturnCodes EventManagerCommon::InitHandler()
             if (!targetApplication.empty()) {
                 bus.sendUnicast(std::make_shared<sevm::BatteryStatusChangeMessage>(), targetApplication);
             }
+            bus.sendUnicast(std::make_shared<sevm::BatteryStatusChangeMessage>(), service::name::appmgr);
```

Comment:
I wonder if it's possible to send it via multicast instead of informing each service separately?

Reply:
Good question ;) Because of some reason `EventManager` works as a dispatcher for this message, i.e. it receives unicast sent from `BatteryController` and then forwards it to other services. I haven't seen such strategy in other places of our OS, so I assumed it's done that way on purpose and didn't change it to prevent introducing potential bugs/regressions, which could appear in both products, as this part is common.

It would be worth analyzing it, though, as these unicasts here are the reason why logs such as "ServiceCellular does not exist" show up on Harmony.

Reply:
Done, reported under MOS-1045 ticket!